tests/common.go

```diff
@@ -3916,6 +3921,7 @@ func ParseFlags() {
 	flag.IntVar(&testsetID, testSetIDFlag, 0, "testset id to post the results")
 	flag.StringVar(&testBranch, testBranchFlag, "master", "branch of the product")
 	flag.StringVar(&testProduct, testProductFlag, "PxEnp", "Portworx product under test")
+	flag.StringVar(&pxRuntimeOpts, "px-runtime-opts", "", "run time options for cluster update")
```

Comment:
It seems, this parameter support passing multiple runtime options, can we change the description with more information like `comma separated list of runtime options`

Reply:
fixed